FRAuth/FRAuth/Callbacks/SingleValueCallback.swift

```diff
@@ -14,7 +14,7 @@ import Foundation
  SingleValueCallback is a base Callback implementation that has single user input value. Any Callback that accepts single value from user interaction without OpenAM's validation with policies may inherit from this class.
  */
 @objc(FRSingleValueCallback)
-public class SingleValueCallback: Callback {
+open class SingleValueCallback: Callback {
```

Comment:
Update year on the header for the modified classes

Reply:
Updated